pwa/src/components/handlers/handlerTable.tsx

```diff
@@ -40,19 +39,12 @@ export default function HandlerTable({ id }) {
       .catch((error) => {
         setShowSpinner(false);
         console.log("Error:", error);
```

Comment:
We should never `console.log` anything in our codebase.

This should be a new error: `Throw new Error (...)`